### DISCUSSION TITLE
feature request: web ui file manager

### DISCUSSION THREAD

**Discussion by FONZACUS** [community/community#22274] [New to GitHub]
<p>ey guys, newish here, long time lurker.</p>
<p>is it possible to have a web based file manager in the browser? something basic like google drive where we can do basic file/folder operations without digging too deep. for example, being on a pc without git installed, or even worse, on mobile. a basic terminal with common commands would be a nice starting point too.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/fonzacus">@fonzacus</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list.</p>
<p>Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by aldass** [comment]
<p>As an additional UI feature request could you add the <strong>&lt;clipboard-copy&gt;</strong> github component any place there is a branch name?  Especially in the <a href="//github.com/%7Borg%7Cuser%7D/%7Brepo-name%7D/" rel="nofollow noopener">//github.com/{org|user}/{repo-name}/</a><strong>branches/</strong> section of github?</p>
